website/partners/api/v2/filters.py

```diff
@@ -56,3 +57,30 @@ def get_schema_operation_parameters(self, view):
                 "schema": {"type": "number"},
             }
         ]
+
+
+class VacancyCategoryFilter(filters.BaseFilterBackend):
+    """Allows you to filter by category slug."""
+
+    def filter_queryset(self, request, queryset, view):
+        categories = request.query_params.get("categories", None)
+
+        if categories:
+            categories_set = VacancyCategory.objects.filter(
+                slug__in=categories.split(",")
+            ).all()
+
+            queryset = queryset.filter(categories__in=categories_set)
```

Comment:
```suggestion
            queryset = queryset.filter(categories__slug__in=categories.split(","))
```

This does not work?

Reply:
I don't think so, because categories is a Many2Many field, so there is no `categories.slug`. I only looked online though, did not actually try that way, but I'll give it a try now.

Reply:
Oh very cool, that does seem to work!